content.md

```diff
@@ -1471,8 +1517,10 @@ cast is cast either "for" or "against" the referendum, and a Methuselah
 may cast some of their votes and ballots in favour and some against, as
 they choose, with the restriction that all of the votes and ballots from
 any given vampire or other source must be cast in agreement as a group.
-Once cast, a vote or a ballot cannot be changed. Methuselahs can gain
-votes and ballots in the following ways:
+Once cast, a vote or a ballot cannot be changed. 
+
+
+Methuselahs can gain votes and ballots in the following ways:
```

Comment:
I would be careful with changes carriage returns and stuff like that. I have no idea how the Markdown format get translated back to HTML or PDF, but this might impact the overall layout in an unexpected way.

Reply:
The PDF rulebook layout is updated manually. The markdown layout only applies to the markdown version (even though it is usually translated somehow to other layouts).